monocle/github/pullrequest.py

```diff
@@ -258,6 +268,11 @@ def get_one(self, org, repository, number):
         return (raw, self.extract_objects([raw]))
 
     def extract_objects(self, prs):
+        def get_login(data):
+            if data and 'login' in data and data['login']:
+                return data['login']
+            return 'ghost'
```

Comment:
It is fine to have author at None.

Reply:
this is the github account that is used for deleted users